Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md

```diff
@@ -0,0 +1,3 @@
+#### Playbooks
+##### New: File Analysis - ReversingLabs TitaniumCloud
+- Check for sample analysis results on A1000. If the sample does not exist on A1000, upload sample to ReversingLabs A1000 appliance and automatically retrieve full & classification reports. Calculate final classification.
```

Comment:
Release notes should be in Pack Release Notes Format.[ For more information please see: ](https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.).
Also, can you explain about uploading a sample if it does not exist on A100? does the playbook uploading it or is the user needs to?
```suggestion
- Created a new playbook that checks for sample analysis results on A1000. If the sample does not exist on A1000, upload the sample to ReversingLabs A1000 appliance and automatically retrieve full & classification reports. Calculate final classification.
```

Reply:
If the sample does not exist on A1000, the playbook uploads it.

Reply:
Done